exonum/src/storage/rocksdb.rs

```diff
@@ -14,9 +14,10 @@
 
 //! An implementation of `RocksDB` database.
 use exonum_profiler::ProfilerSpan;
-use rocksdb::TransactionDB as _RocksDB;
+use rocksdb::OptimisticTransactionDB as _RocksDB;
 use rocksdb::DBRawIterator;
-use rocksdb::Snapshot as _Snapshot;
+use rocksdb::optimistic_txn_db::Snapshot as _Snapshot;
+use rocksdb::Transaction as _Transaction;
```

Comment:
I don't like underscore prefixes, but cannot advise a better naming: usually I would suggest to name it `RocksDBTransaction` instead of `_Transaction`, but that name is already taken. :unamused: